src/app/item-popup/DeepsightHarmonizerIcon.tsx

```diff
@@ -11,13 +11,16 @@ export function DeepsightHarmonizerIcon({ item }: { item: DimItem }) {
   const harmonizerIcon = defs.InventoryItem.get(DEEPSIGHT_HARMONIZER)?.displayProperties.icon;
 
   return (
-    <PressTip tooltip={harmonizableTooltipContent(item)} className={styles.deepsightHarmonizerIcon}>
+    <PressTip
+      tooltip={HarmonizableTooltipContent({ item })}
```

Comment:
Very close! Now, to use it as a component, you do:
```suggestion
      tooltip={<HarmonizableTooltipContent item={item} />}
```